services/workflows-service/.env.example

```diff
@@ -16,7 +16,6 @@ API_KEY=secret
 NODE_ENV=local
 SENTRY_DSN=
 WEBHOOK_SECRET=webhook_secret
```

Comment:
We don't need secret either

Reply:
It's for backward compatiable, to not break Fido in production. They will keep using `Authorization` header for a while, but new customers will use `X-HMAC-SIGNATURE`